deployments/helm-chart/templates/controller-prometheus-service.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
 spec:
   clusterIP: None
   ports:
-  - name: prometheus
+  - name: {{ .Values.prometheus.service.endpoint }}
```

Comment:
why do we need the name of the port to be configurable?

Reply:
I made it configurable mostly to make it more clear how this field links to the value of `prometheus.serviceMonitor.endpoints`

When you configure the `endpoints` for a ServiceMonitor to scrape, it uses the name of the port in the service to know which ports to scrape.
So this might look like this:

```yaml
prometheus:
  serviceMonitor:
    endpoints:
      - port: prometheus
```

I wanted to represent the value of the name here as `endpoint` so that it was more obvious how it linked together with the ServiceMonitor `endpoints` field
So for example, in the values yaml it would look like this

Reply:
IMHO we don't need to make it configurable, and the name `prometheus.service.endpoint` is not very clear anyway

Reply:
so from the other thread I think we can revert this change

Reply:
Sounds good.